rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/action/TbCopyAttributesToEntityViewNode.java

```diff
@@ -0,0 +1,137 @@
+                    ctx.getEntityViewService().findEntityViewsByTenantIdAndEntityIdAsync(ctx.getTenantId(), msg.getOriginator());
+            withCallback(entityViewsFuture,
+                    entityViews -> {
+                        List<ListenableFuture<List<Void>>> saveFutures = new ArrayList<>();
+                        for (EntityView entityView : entityViews) {
+                            if ((entityView.getEndTimeMs() != 0  && entityView.getEndTimeMs() > now && entityView.getStartTimeMs() < now) ||
+                                    (entityView.getEndTimeMs() == 0 && entityView.getStartTimeMs() < now)) {
+                                Set<AttributeKvEntry> attributes = JsonConverter.convertToAttributes(new JsonParser().parse(msg.getData())).getAttributes();
+                                List<AttributeKvEntry> filteredAttributes =
+                                        attributes.stream()
+                                                .filter(attr -> {
+                                                    switch (scope) {
+                                                        case DataConstants.CLIENT_SCOPE:
+                                                            if (entityView.getKeys().getAttributes().getCs().isEmpty()) {
+                                                                return true;
+                                                            }
+                                                            return entityView.getKeys().getAttributes().getCs().contains(attr.getKey());
+                                                        case DataConstants.SERVER_SCOPE:
+                                                            if (entityView.getKeys().getAttributes().getSs().isEmpty()) {
+                                                                return true;
+                                                            }
+                                                            return entityView.getKeys().getAttributes().getSs().contains(attr.getKey());
+                                                        case  DataConstants.SHARED_SCOPE:
+                                                            if (entityView.getKeys().getAttributes().getSh().isEmpty()) {
+                                                                return true;
+                                                            }
+                                                            return entityView.getKeys().getAttributes().getSh().contains(attr.getKey());
+                                                    }
+                                                    return false;
+                                                })
+                                                .collect(Collectors.toList());
+                                saveFutures.add(ctx.getAttributesService().save(entityView.getId(), scope, new ArrayList<>(filteredAttributes)));
```

Comment:
I believe we should either:
1) convert the incoming message to multiple outgoing messages with different originators (entity view ids) and let other existing nodes to handle "save to DB and notify to subscriptions"
2) execute ctx.getTelemetryService().saveAndNotify instead of ctx.getAttributesService().save, because dashboards will not get updated in this case.